omnisette/src/lib.rs

```diff
@@ -42,7 +42,7 @@ impl std::fmt::Display for AnisetteMetaError {
 
 impl std::error::Error for AnisetteMetaError {}
 
-pub const DEFAULT_ANISETTE_URL: &str = "https://ani.f1sh.me/";
+pub const DEFAULT_ANISETTE_URL: &str = "https://ani.wesbryie.com/";
```

Comment:
Just want to make sure, did you mean to commit this (your personal anisette v1 server)?

Reply:
I did mean to commit that, but I hadn't realized it was supposed to be a V1 server. It's my personal V3 server. Do you have a V1 server that can be used? F1shys is down

Reply:
I mean, v3 servers respond to v1 style requests, so it should be fine if you're OK with it being your personal server

Reply:
Yeah, that's fine